spec/src/main/asciidoc/architecture.asciidoc

```diff
@@ -1051,21 +1051,29 @@ The Reactive Messaging implementation reads the global application configuration
 
 * `mp.messaging.incoming.[channel-name].[attribute]=[value]`
 * `mp.messaging.outgoing.[channel-name].[attribute]=[value]`
+* `mp.messaging.connector.[connector-name].[attribute]=[value]`
 
 For each extracted `channel-name`:
 
 1. The `type` attribute value is read, and the connector implementation identified. If no connector matches, the deployment must be failed with a `DeploymentException`;
-2. The other attributes are processed to generate a `Config` object containing only `attribute=value` entries.
+2. Relevant attributes are those matching either the `channel-name` or the resolved `connector-name`.
+3. Relevant attributes are processed to generate a `Config` object containing only `attribute=value` entries.
+If is valid to have an attribute specified at a connector level and also for a specific channel.
+If an attribute appears for both a channel and its relevant connector, the connector specific value will be used.
```

Comment:
I would have said otherwise. If the property is defined at the connector level and at the channel level, the channel level wins no? 

Typically, you have a bunch of kafka channels, all but one use the same bootstrap servers, but one of them used another one. That would not be possible with connector > channel.

Reply:
Agh this is just a 'typo' more of a 'braino' as I agree with you!

Reply:
Done.